tests/core/block/e2e/test_correctness.py

```diff
@@ -316,6 +297,11 @@ def test_v1_v2_greedy_equality_prefix_caching_enabled_with_preemption(
 
     NOTE: We want a significant number of generated tokens so that any incorrect
     KV mapping has time to build up error.
+
```

Comment:
nit - Please consider removing reference to block manager v1 in the comment of the test.

Reply:
Changed! TY for suggestion.